backend/routes/XrplNFTHelper.js

```diff
@@ -0,0 +1,462 @@
+
+
+  const client = new xrpl.Client("wss://xrplcluster.com/:51233") 
+  const wallet = xrpl.Wallet.fromSeed(minterKey)
+
+  await client.connect();
+
+  let ticketArr = []
+
+    for (let i=0; i < tickets.length; i++) {
+      ticketArr[i] = tickets[i].TicketSequence;
+    }
+
+    
+    for (let i=0; i < ticketArr.length; i++) {
+      const transactionBlob = {
+          "TransactionType": "NFTokenMint",
+          "Account": wallet.classicAddress,
+          "URI":  xrpl.convertStringToHex("www.testing.com"),
+          "Flags": parseInt(11),
+          "Sequence": 0,
+          "TicketSequence": ticketArr[i],
+          "NFTokenTaxon": 6666
+      }
+      
+      console.log("Minting " + ticketArr[i])
+      const tx =  client.submit(transactionBlob, { wallet: wallet} )
+    }
+
+    let nfts = await client.request({
+      method: "account_nfts",
+      account: this.transactionDetails.Account
+      })
+
+
+    
+    
+  
+    return nfts;
+}
+
+
+async batchX(){
+  try {
+
+
+    const client = new xrpl.Client(this.clientDetails) 
+    const wallet = xrpl.Wallet.fromSeed(this.transactionDetails.Secret) //secret
+    
+    await client.connect()
+
+    const nftokenCount = this.transactionDetails.Memos.numberOfTokens
+
+
+    const account_info = await client.request({
+      "command": "account_info",
+      "account": wallet.classicAddress
+    })
+
+    let my_sequence = account_info.result.account_data.Sequence
+
+    
+    let response = await client.request({
+      "command": "account_objects",
+      "account": wallet.classicAddress,
+      "type": "ticket"
+    })
+
+
+    let tickets = []
+
+    for (let i=0; i < nftokenCount; i++) {
+      tickets[i] = response.result.account_objects[i].TicketSequence
+    }
+
+    
+    const ticketTransaction = await client.autofill({
+      "TransactionType": "TicketCreate",
+      "Account": wallet.classicAddress,
+      "TicketCount": nftokenCount,
+      "Sequence": my_sequence
+    })
+
+
+    
+
+    const signedTransaction = wallet.sign(ticketTransaction)
+    const tx = await client.submitAndWait(signedTransaction.tx_blob)
+
+   
+
+  for (let i=0; i < nftokenCount; i++) {
+    const transactionBlob = {
+        "TransactionType": "NFTokenMint",
+        "Account": wallet.classicAddress,
+        "URI":  this.transactionDetails.URI,
+        "Flags": this.transactionDetails.Flags,
+        "Sequence": 0,
+        "TicketSequence": tickets[i],
+        "NFTokenTaxon": 5555
+    }
+    
+    const tx =  client.submit(transactionBlob, { wallet: wallet} )
+    
+
+    let nfts = await client.request({
+      method: "account_nfts",
+      account: wallet.classicAddress,
+      limit: 400
+    })
+    
+   
+    for(let i = 0; i <= nftokenCount; i++)
+    {
+          nfts = await client.request({
+              method: "account_nfts",
+              account: wallet.classicAddress,
+              limit: 400,
+              marker: nfts.result.marker
+          })
+        
+    }

```

Comment:
This will likely get much more nftokens than you seem to want. I think you may have meant to do something like this to track all the nfts you get, and ensure you request all nfts for the account, even if there are more than 400.
```suggestion
    const limit = 400
    const max_requests = (nftokenCount / limit) * 10 // * 10 since servers do not have to honor the limit you ask for.
    for(let i = 0; i <= max_requests && nfts.length < nftokenCount; i++)
    {
          nfts += await client.request({
              method: "account_nfts",
              account: wallet.classicAddress,
              limit: 400,
              marker: nfts.result.marker
          })
    }
```